### ISSUE TITLE
The "WCAG20_Frame_HasTitle" should check descriptive labels in addition to the title #3795

### ISSUE THREAD

**Issue by aliunwala** [IBMa/equal-access#918]
The original check is based on [WCAG21 Techniques H64](https://www.w3.org/WAI/WCAG21/Techniques/html/H64) to require a title attribute for an iFrame element. However, the aria labels can also be applied to fulfill the same purpose, especially when the iframe with a 'document' or 'application' role, an aria-labeledby or aria-describedby may be more appropriate than a simple title.  
  Suggest reporting violation only if no aria label exists.  

## This ticket is for the following (select only one): 

- [ ] `New test cases`
- [ ] `Update to a current rule`
- [ ] `Helping text for a rule`
- [ ] `other`

## Provide test file location: 

![image](https://user-images.githubusercontent.com/4451128/171962402-5ab18f6e-f95b-4071-997e-7440339468eb.png)

**Comment by aliunwala** [comment]
Triage:
@shunguoy Will investigate if this is still valid

**Comment by shunguoy** [comment]
fixed during the engine refactoring though the rule name now doesn't reflect the rule well #2055
